### DISCUSSION TITLE
Regarding to lang

### DISCUSSION THREAD

**Discussion by Soebb** [oliverfei/videocr-PaddleOCR#3] [Q&A]
Hi and thanks for this awesome repo,
Recently I got this error for 'fa' (Arabic related lang) :

AssertionError: param lang must in dict_keys(['ch', 'en', 'french', 'german', 'korean', 'japan']), but got fa

It seems the default model doesn't work for 'fa' and i need to download another model and add its path to the det_model_dir/rec_model_dir, am i right?
Does the downloaded compressed model itself contain the params?
I read the Readme but still confused.

**Comment by devmaxxing** [comment]
Hello, what version of PaddleOCR are you using?
With the latest PaddleOCR release, when I tried specifying lang as 'fa' without specifying any models, the program automatically downloaded the default arabic model and seems to work without error. They do also appear to have an 'fa' specific model listed here: PaddlePaddle/PaddleOCR#1048

**Reply by Soebb** [reply]
Recently i wanted to test it in local system by a [telegram bot](https://github.com/Soebb/vse) (because i only tested repo on heroku before) but i get this error :

![image_2021-12-04_00-04-35](https://user-images.githubusercontent.com/87156166/144679839-264857f0-d5a8-433f-9720-59ef0862d202.png)
Windows 10 64Bit
Python 3.10

Is paddlepaddle support Pythons3 greater than 3.8 ? Because not mentioned in its github readme.

**Reply by devmaxxing** [reply]
It looks like they just support 3.8 for now: https://pypi.org/project/paddlepaddle/
